tabs/configuration.js

```diff
@@ -157,10 +157,10 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             );
         }
 
-        if (CONFIG.flightControllerIdentifier == "BTFL" && semver.gte(CONFIG.flightControllerVersion, "2.8.0")) {
+        if (CONFIG.flightControllerIdentifier === "BTFL" && semver.gte(CONFIG.flightControllerVersion, "2.8.0")) {
              features.push(
                 {bit: 22, group: 'other', name: 'AIRMODE'},
-                {bit: 23, group: 'other', name: 'SUPEREXPO_RATES'}
+                {bit: 23, group: 'other', mode: 'readonly', name: 'SUPEREXPO_RATES', haveTip: true}
```

Comment:
> I bet that this will raise many questions....why cant I change it :(

There is a help text in there explaining it. Most users will have used the cleanflight configurator in the past, and will go there to find the SUPEREXPO_RATES feature.

If you think it should be out of there, can you please remove the rest of the changes that I've made to accommodate it as well?